woocommerce/woocommerce-android
# Issue/6934 payments alert dialog compose UI

Closes: #6934 


### Description

This PR adds Jetpack Compose UI for alert dialog that is shown when trying to dismiss Upsell Card reader Banner. This PR doesn't add any business logic for the Alert dialog apart from the UI.

### Testing instructions

The code isn't integrated with the Android views in this PR. You can navigate to the `BannerDismissDialog.kt` file and see the preview in the Android studio.

Or, you can build [this PR](https://github.com/woocommerce/woocommerce-android/pull/6984) and see how the dismiss alert dialog looks in action.

### Screenshot

<img src=https://user-images.githubusercontent.com/1331230/179727894-fb8a67c6-055e-4190-aac8-b0a3cbaa6a9f.png width=350px height=800px />
